cime_config/config_component.xml

```diff
@@ -65,8 +65,8 @@
     <type>char</type>
     <valid_values>isopyc_bulkml,cntiso_hybrid</valid_values>
     <default_value>isopyc_bulkml</default_value>
-    <group>run_component_blom</group>
-    <file>env_run.xml</file>
+    <group>build_component_blom</group>
```

Comment:
Thanks for fixing this.

Reply:
Does this mean that hybrid coordinates would not work in release v.1.4.0?

Reply:
It should work - my comment is only that the xml variable BLOM_VCOORD should be in env_build.xml and not in env_run.xml - since it sets dimensions.
That said - I really think we need to expand the test list to include  hybrid coordinates and more testing of OMP and MPI tasks variations.

Reply:
Hybrid coordinates worked in v1.4.0 by modifying BLOM_VCOORD in env_run.xml, but you would have to remember to do it before building.